pkg/pod/status.go

```diff
@@ -281,7 +281,7 @@ func getFailureMessage(pod *corev1.Pod) string {
 	for _, status := range pod.Status.ContainerStatuses {
 		term := status.State.Terminated
 		if term != nil && term.ExitCode != 0 {
-			return fmt.Sprintf("%q exited with code %d (image: %q); for logs run: kubectl -n %s logs %s -c %s",
+			return fmt.Sprintf(`%q exited with code %d (image: %q); for logs run: kubectl -n %s logs %s -c %s\n`,
```

Comment:
would be great to include a comment here like

```
// Newline escape required at end to prevent yaml parser from breaking the log help text at 80 chars
```

Reply:
I have updating with the comment @sbwsg